src/databricks/labs/ucx/installer/policy.py

```diff
@@ -0,0 +1,94 @@
+import json
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterPolicyInstaller:
+    def __init__(self, installation: Installation, ws: WorkspaceClient):
+        self._ws = ws
+        self._installation = installation
+        self._policy_id = ""
+
+    @staticmethod
+    def _policy_config(value: str):
+        return {"type": "fixed", "value": value}
+
+    def create_cluster_policy(
+        self, inventory_database: str, spark_conf: dict, instance_profile: str | None
+    ) -> str | None:
+        policy_name = f"Unity Catalog Migration ({inventory_database}) ({self._ws.current_user.me().user_name})"
+        policies = self._ws.cluster_policies.list()
+        policy_id = None
+        for policy in policies:
+            if policy.name == policy_name:
+                policy_id = policy.policy_id
+                logger.info(f"Cluster policy {policy_name} already present, reusing the same.")
+                break
+        if not policy_id:
+            logger.info("Creating UCX cluster policy.")
+            policy_id = self._ws.cluster_policies.create(
+                name=policy_name,
+                definition=self._cluster_policy_definition(conf=spark_conf, instance_profile=instance_profile),
+                description="Custom cluster policy for Unity Catalog Migration (UCX)",
+            ).policy_id
+        assert policy_id is not None
+        self._policy_id = policy_id
+        return self._policy_id
```

Comment:
```suggestion
        logger.info("Creating UCX cluster policy.")
        return self._ws.cluster_policies.create(
            name=policy_name,
            definition=self._cluster_policy_definition(conf=spark_conf, instance_profile=instance_profile),
            description="Custom cluster policy for Unity Catalog Migration (UCX)",
        ).policy_id
```
avoid nesting as much as possible